src/Illuminate/Foundation/Testing/Concerns/InteractsWithDatabase.php

```diff
@@ -8,7 +8,9 @@
 use Illuminate\Support\Arr;
 use Illuminate\Support\Facades\DB;
 use Illuminate\Testing\Constraints\CountInDatabase;
+use Illuminate\Testing\Constraints\emptyDatabase;
```

Comment:
The use also needs to be fixed

Reply:
thought they were refactored after I re-imported the class with the new name!